### ISSUE TITLE
proprietary components

### ISSUE THREAD

**Issue by IzzySoft** [TangoBeee/Weathernaut#13]
Looks like today's release made your app non-free by introducing Google Mobile Services. Was that intentional? Hm, looks like: the culprit can be found [here](https://github.com/TangoBeee/Weathernaut/blob/master/app/build.gradle.kts#L67):

```gradle
    // Google PlayService Location Lib
    implementation("com.google.android.gms:play-services-location:21.2.0")
```

Is that really necessary – or would you consider reverting that? For now, we need to add the `NonFreeComp` anti-feature to your app at IzzyOnDroid – but that can of course be reverted once there's no proprietary component included anymore.

**Comment by droidv3** [comment]
IS this something same as Advertisement ID cause playstore seem to think so ?

**Comment by IzzySoft** [comment]
No, that would be something different (and require the `Ads` anti-feature then). But we didn't see an ad framework in the APK.
